modules/upgrade-upgrading.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrading-aws.adoc
+
+[id='upgrade-upgrading-{context}']
+= Updating a cluster
+
+If updates are available, you can update your cluster from the web console.
+
+.Prerequisites
+
+* Access to the web console as a user with `admin` privileges.
+
+.Procedure
+
+. From the web console, click *Administration* > *Cluster Settings* and review
+the contents of the *Overview* tab.
+.. For production clusters, ensure that the *CHANNEL* is set to `stable-4.0`.
+.. If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
+cluster.
+.. The *DESIRED VERSION* indicates the cluster version that your cluster is running
+or is updating to.
+
+. Click *Updates Available*, select a version to update to, and click *Update*.
+The *UPDATE STATUS* changes to `Updating`, and you can review the progress of
+the Operator upgrades on the *Cluster Operators* tab.
+
+
```

Comment:
@smarterclayton do you docs on how to use `oc adm upgrade` to achieve same results?

Reply:
The docs team is having ongoing discussions about whether to prefer the UI or CLI workflows or if we need to document both, as required. If you want the `oc adm upgrade` flow documented, I can start a draft for it. I'll need information about how a GA user should know the valid release images and if it'll differ for OCP and OKD.

Reply:
The same question here, if we will have some steps on how to do upgrade through CLI.

Reply:
I think we should provide CLI and UI flows (as not all users are alike), your UI users will be UI users, but your CLI users may also us the UI some of the time. 

Starting with the UI is fine, but customers who want more power and control will switch to the CLI as they become familiar and begin automating steps.

Reply:
That's fair @sferich888. I'm happy to document the CLI flow as soon as update's working as intended.